docs/guides/main.rst

```diff
@@ -290,6 +290,36 @@ You can always get help using the ``beet help`` command. The plain ``beet help``
 command lists all the available commands; then, for example, ``beet help
 import`` gives more specific help about the ``import`` command.
 
+Beets Core Concepts
+-------------------
+
+Beets relies on a few core concepts that underlie every feature and every plugin you'll use.
+Understanding these common pieces can help you see how to harness their full power.
+
+* **The library.** The beets *Library* is a database that catalogs your music.
+  The Library does not contain any music files itself;
+because it's a database, it saves the paths (i.e., the filenames)
```

Comment:
Right; as pointed out by Travis: https://travis-ci.org/beetbox/beets/jobs/213252579#L885

(This may be my fault.)